dumbot.py

```diff
@@ -426,16 +421,17 @@ def add_cmd(to):
                     add_cmd(getattr(item, name))
 
     def run(self):
-        if self._loop.is_running():
```

Comment:
Was this deprecated?

Reply:
[`loop.is_running()`](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.is_running) isn't deprecated, but passing loop objects around is.